app/views/main.scala.html

```diff
@@ -46,5 +46,7 @@
 
         @fragments.global.footer(product, edition, contactUsCountry)
         @fragments.footerJavaScript()
+        <!--AWIN affiliate tracking tag - see here: https://drive.google.com/drive/folders/19AOGPkPbbFjqPJTdCcCba3EwpLiSHBwW -->
+         <script src="https://www.dwin1.com/11152.js" type="text/javascript" defer="defer"></script>
```

Comment:
should this only be referenced on the thank you page when the variables are set?